parts/configure-swarmmode-cluster.sh

```diff
@@ -150,9 +151,13 @@ echo "Installing and configuring Docker"
 installDocker()
 {
   for i in {1..10}; do
-    wget --tries 4 --retry-connrefused --waitretry=15 -qO- https://get.docker.com | sh
+    curl --max-time 60 -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - 

```

Comment:
my bad I just saw SWARM_VERSION="swarm:1.1.0" and assumed that was pinning. Guess it is left over.